include/logger.h

```diff
@@ -41,14 +41,18 @@ class CoreExport FileWriter
 	 */
 	FILE* log;
 
+	/** The number of write operations after which we should flush.
+	 */
+	int flush;
```

Comment:
@del6597 Thank you for the input, it indeed can create problems but IMO the solution in this case is to fix the remaining code to use unsigned integers as well, not to increase the amount of code using wrong types.